backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -61,6 +61,8 @@ case class DefaultStandardAsyncExecutionActorParams
   override val minimumRuntimeSettings: MinimumRuntimeSettings
 ) extends StandardAsyncExecutionActorParams
 
+case class ScriptPreambleData(bashString: String, executeInSubshell: Boolean)
```

Comment:
- Case classes can have defaults
- Introducing a default gives a good opportunity to explain why it is what it is
```suggestion
// Typically we want subshell for encapsulation
// Override to `false` when we want the script to set an environment variable in the parent shell
case class ScriptPreambleData(bashString: String, executeInSubshell: Boolean = true)
```